library/network/src/modules/SuSEFirewallProposal.rb

```diff
@@ -781,6 +779,8 @@ def propose_iscsi
 
       # bsc#916376: ports need to be open already during boot
       SuSEFirewall.full_init_on_boot(true)
+
```

Comment:
That's described here: https://github.com/yast/yast-yast2/blob/SLE-12-GA/library/network/src/modules/SuSEFirewall.rb#L3735-L3739